scripts/create-component/templates/storybook/{{componentName}}.stories.tsx.hbs

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { {{componentName}}, {{componentName}}Props } from '@fluentui/{{packageName}}';
+// import { PartialTheme, ThemeProvider } from '@fluentui/react-theme-provider';
```

Comment:
remove ?

Reply:
This will break the scaffolding of the storybook examples, the commented line is also there to aid implementation of the themes quicker. Can you give more guidance on reasoning ?

Reply:
To clarify - I meant to remove the commented out line.
I don't think individual stories need `ThemeProvider` as the `ThemeProvider` is wrapped by storybook decorator `withThemeProvider`. 

In fact, we should modify [preview.js](https://github.com/microsoft/fluentui/blob/master/packages/react-examples/.storybook/preview.js#L29) to ensure packages for converged component are covered. Instead of having a white list of package names, perhaps we should by default add the decorator for all `react-*` packages and have a black list for exceptions

Reply:
Agreed, and I actually think this whole block:
https://github.com/microsoft/fluentui/blob/82d09c561d1588c03d72f7715dbf86f668944d0d/packages/react-examples/.storybook/preview.js#L16-L31

Should be replaced with this:
```js
if (!['react-charting', 'react-experiments'].includes('PACKAGE_NAME')) {
  addDecorator(withStrictMode);
}
if (!['react-theme-provider'].includes('PACKAGE_NAME')) {
  addDecorator(withThemeProvider);
}
```
In v8, the strict mode decorator is okay for all packages except charting and experiments. 

@xugao can correct me if I'm wrong, but I'm guessing the ThemeProvider decorator should be harmless for older packages, but should probably not be applied for the theme provider package itself.

For reference, this is the full list of package which DO NOT have the theme provider decorator today:
azure-themes
react
react-charting
react-date-time
react-experiments
react-focus
react-theme-provider

Reply:
Updated

Reply:
Updated and added issue [#16646](https://github.com/microsoft/fluentui/issues/16646) for triage